src/csrc/rl/off_policy/sac.cpp

```diff
@@ -179,6 +218,15 @@ SACSystem::SACSystem(const char* name, const YAML::Node& system_node,
     THROW_INVALID_USAGE("Missing critic_lr_scheduler block in configuration file.");
   }
 
+  // alpha parameter
+#if 0
```

Comment:
Remove unused code block here if not needed.

Reply:
That one is actually needed, I removed the comment.